.ci/azure-pipelines-test.yml

```diff
@@ -94,5 +94,5 @@ jobs:
         displayName: 'Publish OpenAPI Artifact'
         condition: and(succeeded(), eq(variables['Agent.OS'], 'Linux'))
         inputs:
-          targetPath: "tests/Jellyfin.Api.Tests/bin/Release/net5.0/openapi.json"
+          targetPath: "tests/Jellyfin.Server.Integration.Tests/bin/Release/net5.0/openapi.json"
```

Comment:
would prefer `Jellyfin.Server.Tests.Integration` and maybe `Jellyfin.Server.Tests.Unit`

Reply:
or just `Jellyfin.Tests.Integration` etc